src/actions/articleActions.js

```diff
@@ -62,16 +65,15 @@ export const fetchComments = article => dispatch => {
 
 export const fetchArticles = () => dispatch => {
   dispatch(getArticlesInitiated(true));
-  return axiosInstance
+  axiosInstance
     .get('/api/article/')
     .then((response) => {
       dispatch(getAllArticles(response.data.article));
     });
 };
 
 export const fetchSpecificArticle = slug => dispatch => {
-  dispatch(getArticlesInitiated(true));
-  return axiosInstance
```

Comment:
Why did you remove both lines of code?

Reply:
The `getArticlesInitiated(true)` was not adding anything to the code. Instead was disabling the loading of the list of articles. This was removed. Without any other statement in the arrow function, there is no need for the return statement.

Reply:
When you fetch a new article and it takes longer, what does the user see while that happens?

Reply:
I added a loader to show as the article is being fetched